src/scripts/generate-file.ts

```diff
@@ -0,0 +1,7 @@
+import fs from "fs";
+import prettier from "prettier";
+
+export const generateFile = (filepath: string, content: string) => {
+  const output = prettier.format(content, { filepath });
```

Comment:
I think you need to add a `parser` option, e.g. compare 

https://github.com/octokit/plugin-rest-endpoint-methods.js/blob/53e3403d39e35b25ac3b5ae872895ab4277067cd/scripts/update-endpoints/types.js#L289-L310

Reply:
> I think you need to add a `parser` option, e.g. compare
> 
> [octokit/plugin-rest-endpoint-methods.js:scripts/update-endpoints/types.js@`53e3403`#L289-L310](https://github.com/octokit/plugin-rest-endpoint-methods.js/blob/53e3403d39e35b25ac3b5ae872895ab4277067cd/scripts/update-endpoints/types.js#L289-L310)

Seems that prettier [infers it](https://prettier.io/docs/en/options.html#parser) from the file extension by default. If you want I can add it as an optional parameter on this utility function :)

Reply:
Resolving comment @gr2m ;)